Plugin/Source/GUI/ModulatableSlider.h

```diff
@@ -16,13 +16,17 @@ class ModulatableSlider : public foleys::AutoOrientationSlider,
 
     using PluginEditorCallback = std::function<juce::AudioProcessorEditor*()>;
     void setPluginEditorCallback (PluginEditorCallback&& newCallback);
+    
+    friend class ModSliderItem;
```

Comment:
Wasn't sure what the best option here was, if having a friend class for the slider or making a public method available to call on to call `StartTimerHz(24)`, as `StartTimerHz()` is a private method

Reply:
Hmm, I think a cleaner fix would be to make ModulatableSlider inherit from Timer publicly.

Reply:
Ah I didnt catch that thanks, much simpler.